File: Python/project/Encryption.py
import random
import string


def encryption(sentence=""):
    letter = list(string.ascii_lowercase)
    punctuation = list(string.punctuation)
    value = ""
    for i in sentence:
        if i.lower() in letter:
            x = random.randint(0, 25)
            value = value + letter[x]
        elif i in punctuation:
            x = random.randint(0, 31)
            value = value + punctuation[x]
        else:
            value = value + " "
    return value


sentence = input("Enter a sentence or phrase: ")
print(sentence)
sentence = encryption(sentence)
print("1:", sentence)
sentence = encryption(sentence)
print("2:", sentence)
sentence = encryption(sentence)
print("3:", sentence)

File: Java/JavaClass/src/com/sri/course/Encryption.java
package com.sri.course;

import java.util.*;

public class Encryption {
    public static void main(String[] args) {
        Scanner oldScanner = new Scanner(System.in);
        System.out.println("Please enter a sentence or phrase. ");
        String sentence = oldScanner.nextLine();
        sentence = encrypted(sentence);
        System.out.println("1: " + sentence);
        sentence = encrypted(sentence);
        System.out.println("2: " + sentence);
        sentence = encrypted(sentence);
        System.out.println("3: " + sentence);
    }

    public static String encrypted(String input){
        String encrypted = "";
        ArrayList<Character> letters = new ArrayList<>();
        for(char i = 'a'; i <= 'z'; i++) {
            letters.add(i);
        }
        char[] punctuation = "!\\\"#$%&'()*+,-./:;<=>?@[\\\\]^_`{|}~\"".toCharArray();
        for(int i=0;i<input.length();i++){
            if(letters.contains(Character.toLowerCase(input.charAt(i)))){
                int x = (int) (25 * Math.random());
                encrypted = encrypted + letters.get(x);
            }else{
                if(input.charAt(i) != ' '){
                    int x = (int) (35 * Math.random());
                    encrypted = encrypted + punctuation[x];
                }else{
                    encrypted = encrypted + " ";
                }

            }
        }
        return encrypted;
    }//end encrypted method
}
